clients/da-vinci-client/src/main/java/com/linkedin/davinci/DaVinciBackend.java

```diff
@@ -252,6 +252,7 @@ public DaVinciBackend(
           .map(objectCacheConfig -> new ObjectCacheBackend(clientConfig, objectCacheConfig, schemaRepository));
 
       ingestionService = new KafkaStoreIngestionService(
+          storageService,
           storageService.getStorageEngineRepository(),
```

Comment:
do we need both ? if we pass the storage service, shouldnt the repo be extracted internally ?